### PR TITLE
Small fixes and typos

### PR DESCRIPTION
**Pull request by lucach**

### CODE REVIEW

File: mini_sections/simple_git.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ The following recommendations help you to get the most out of git.
 They assume you know how to work with git and in particular with different branches.
 If you don't know git well and don't want to learn it now, just ignore this page.
 
-1. **Keep the `master` branch always clean:** Make sure code on the master branch always compiles 
+1. **Keep the `main` branch always clean:** Make sure code on the master branch always compiles
```
Comment (MiWeiss) [inline mini_sections/simple_git.md:7]:
```suggestion
1. **Keep the `main` branch always clean:** Make sure code on the main branch always compiles 
```

File: mini_sections/simple_git.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ The following recommendations help you to get the most out of git.
 They assume you know how to work with git and in particular with different branches.
 If you don't know git well and don't want to learn it now, just ignore this page.
 
-1. **Keep the `master` branch always clean:** Make sure code on the master branch always compiles 
+1. **Keep the `main` branch always clean:** Make sure code on the master branch always compiles 
     and does not break any functionality that worked in previous versions.
     Like that, you have always a version ready to submit for milestone discussions.
 2. **Implement new features on distinct branches** and merge them into master when the feature is completely implemented.
```
Comment (MiWeiss) [inline mini_sections/simple_git.md:10]:
```suggestion
2. **Implement new features on distinct branches** and merge them into main when the feature is completely implemented.
```

**Review by MiWeiss — CHANGES_REQUESTED** [review]
Thanks for spotting them! I added two more `master` => `main`.
Can you access my suggestions (it appears I do not have push access to your branch)?

I'll then merge

**Comment by lucach** [comment]
Done!

**Pull request merged**